adding schnitzel viennese style (#40)
SUSE/recipebook: recipes/main/schnitzel_viennese_style.adoc

[id='sec.schnitzel_viennese_style']

ifdef::env-github[]
:imagesdir: ../../images
endif::[]
ifndef::env-github[]
:imagesdir: images
endif::[]

=== Schnitzel Viennese Style

image::schnitzel_viennese_style/schnitzel.jpg[Name, 640, 480]

Contributed by: Robert
image:contributors/robert_s.png[Name, 140, 140]

> Schnitzel is very common in Germany, the original however is from Austria. Schnitzel Viennese (Wiener Schnitzel) has to be made of veal, the more popular made of pork (rarely turkey) is called Viennese Style (Wiener Art). Making is very much the same and since it's crumbed, there's usually no sauce with it.

Reviewer: ```suggestion
Schnitzel is very common in Germany, the original however is from Austria. Authentic Schnitzel Viennese (Wiener Schnitzel) has to be made from veal, the more popular variation made from pork (rarely turkey) is called Viennese Style (Wiener Art). Making is very much the same and since it's coated in breadcrumbs, there's usually no sauce with it.
```

Author: thx for the suggestions, mostly adjusted...

Ingredients (4 servings):

[width="100%",cols="3,3,10,10",options="header"]
|=========================================================
|Metric | US    | Ingredient         | Zutat

| 4     |       | pork escalopes     | Schweineschnitzel
|       |       | starch flour       | Stärkemehl 
| 2     |       | eggs               | Eier 
|       |       | breadcrumbs        | Semmelbrösel
|       |       | salt               | Salz
|       |       | pepper             | Pfeffer 
|       |       | resolidified butter| Butterschmalz
| 1     |       | lemon              | Zitrone
|=========================================================

Preparation:

> . Dry the schnitzels with a paper towel, put them on a wooden cutting board and treat them with a meat tenderiser, as thin as you like them. Or as big as your pan is.

Reviewer: ```suggestion
. Pat the meat dry with a paper towel, put the pieces on a wooden cutting board and pound them out with a meat tenderizer, as thin as you like them. Or as big as your pan is.
```

> . Prepare three plates or bowls, first with with starch flour, for the second mix the eggs and add a good amount of salt and pepper, the third gets the breadcrumbs.

Reviewer: ```suggestion
. Prepare three plates or bowls, first with with starch flour, scramble the eggs into the second and add a good amount of salt and pepper (to taste), the third gets the breadcrumbs.
```

> . Pull the schnitzels first through the starch so they are covered and the egg will better stick.

Reviewer: ```suggestion
. Put the schnitzels first into the starch so they are covered and the egg will stick better.
```

Author: partly adjusted ...

> . Pull the schnitzels through the eggs, they must be fully covered, no dry spots.

Reviewer: ```suggestion
. Put the schnitzels into the egg mixture, they must be fully covered, with no dry spots remaining.
```

Author: pull, not put :-)

> . Pull the schnitzels through the breadcrumbs and press on, cover them completely. 

Reviewer: ```suggestion
. Put the schnitzels into the last bowl and press onto them, covering them completely in breadcrumbs. 
```

Author: I actually pull them through, not put them in :-)

> . Heat a big enough pan with plenty resolidified butter, fry the schnitzels at medium heat (not too hot) a few minutes until they are well done and nicely browned, turn them.

Reviewer: ```suggestion
. Heat a big enough pan with plenty resolidified butter, fry the schnitzels at medium heat (not too hot) a few minutes until they are well done and nicely browned, flip and repeat.
```

Author: partly adjusted ...

Keep them warm in the oven for not too long and serve the schnitzels with lemon sliced or quartered so everyone can sour them at own taste.

> Schnitzel goes well with potato salad (mayonnaise, or franconian style with vinegar and oil) or as children love it with french fries. Also popular to take along (cold), put them on a french baguette as a hoagie, along with cucumber, tomatoes, fried onions and ketchup/mayonnaise.

Reviewer: ```suggestion
Schnitzel goes well with potato salad (mayonnaise, or franconian style with vinegar and oil) or as children love it with french fries. They are also popular to take them with you (once cooled), put them on a french baguette as a hoagie, along with cucumber, tomatoes, fried onions and ketchup/mayonnaise.
```

Author: thx, I like it the way it is.
